adapta/security/clients/_azure_client.py

```diff
@@ -196,3 +196,9 @@ def select_file_system(options: Optional[Dict[str, str]], account_name: str) ->
         return SubTreeFileSystem(
             path.to_hdfs_path(), PyFileSystem(FSSpecHandler(select_file_system(connection_options, path.account)))
         )
+
+    def initialize_session(self, session_callable=None) -> "AzureClient":
```

Comment:
Missing type hints

Reply:
Missing indeed!
Added in [3943e2d](https://github.com/SneaksAndData/adapta/pull/437/commits/3943e2dda0291afba169f072092af8a0ff6741b7)
Thank you for pointing it out 🙂